conan/internal/api/new/barebones.py

```diff
@@ -0,0 +1,11 @@
+_conanfile = '''\
+from conan import ConanFile
+
+class BarebonesConanfile(ConanFile):
+    name = "barebones"
```

Comment:
Isn't the literal "barebones" name a bit unexpected? The ``cmake_lib`` is not called ``cmake_lib/1.0``. Maybe something neutral like ``pkg``?